PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEmcalJetSpectra8TeVTriggerQA.cxx

```diff
@@ -366,86 +398,118 @@ void AliAnalysisTaskEmcalJetSpectra8TeVTriggerQA::DoJetLoop()
     TString groupname;
     AliJetContainer* jetCont = 0;
     TIter next(&fJetCollArray);
-
-	TClonesArray *triggerpatches = dynamic_cast<TClonesArray *>(InputEvent()->FindListObject("EmcalTriggers"));
-	if(!triggerpatches)
-		AliErrorStream() << "Trigger patch container EmcalTriggers not found in task " << GetName() << std::endl;
-
+    
+    TClonesArray *triggerpatches = dynamic_cast<TClonesArray *>(InputEvent()->FindListObject("EmcalTriggers"));
```

Comment:
I believe @mfasDa had a comment here on your previous PR. I think it was because he expected the EMCal framework to provide the triggers (?)